quantus/metrics/robustness/max_sensitivity.py

```diff
@@ -339,6 +353,13 @@ def evaluate_batch(
 
             # Measure similarity for each instance separately.
             for instance_id in range(batch_size):
+                if (
+                    self.return_nan_when_prediction_changes
+                    and instance_id in changed_prediction_indexes
+                ):
+                    similarities[instance_id, step_id] = np.nan
+                    continue
```

Comment:
Is `continue` really necessary here? since instance_id is an `int`?

Reply:
Yes, this `continue` skips the computation of `sensitivities` for an instance, if the prediction changed.